tot/models.py

```diff
@@ -225,3 +231,302 @@ def maybe_drop_added_dates(self, predicted, df):
             predicted_new, received_ID_col_pred, received_single_time_series_pred, received_dict_test_pred
         )
         return predicted, df
+
+
+@dataclass
+class SeasonalNaiveModel(Model):
+    """
+    A `SeasonalNaiveModel` is a naive model that forecasts future values of a target series based on past observations
+    of the target series of the specified period, i.e season.
+
+    The naive models act on time series data having ``fit()`` and ``predict()`` methods. The season can be specified
+    by K steps. Therefore, the model predicts the value of `K` time steps ago. When `K>1`, it repeats the last `K`
+    values.
+
+    Parameters
+    ----------
+        K : int
+            the number of last time steps to repeat
+        n_forecasts : int
```

Comment:
do we need to define this here, or is this because of framwork expectations?

Reply:
@ourownstory I have specified the model input variables here to have an overview of the variables that can be given as input. Would you rather define them somewhere else? Happy to have your recommendation!